native/src/main/java/io/ballerina/stdlib/file/nativeimpl/Utils.java

```diff
@@ -266,7 +266,11 @@ public static Object copy(BString sourcePath, BString destinationPath, BString..
                 Files.walkFileTree(srcPath, new RecursiveFileCopyVisitor(srcPath, destPath, ops));
             }
         } catch (IOException ex) {
-            return FileUtils.getBallerinaError(FileConstants.FILE_SYSTEM_ERROR, ex);
+            return FileUtils.getBallerinaError(FileConstants.FILE_SYSTEM_ERROR,
+                    "An error occurred when copying the file/s: " + ex);
+        } catch (NullPointerException ex) {
+            return FileUtils.getBallerinaError(FileConstants.FILE_NOT_FOUND_ERROR,
+                    "The target directory does not exist: " + ex);
```

Comment:
Do we need ex in the error message? can we avoid this null pointer exception

Reply:
Avoided in [c2f3749](https://github.com/ballerina-platform/module-ballerina-file/pull/430/commits/c2f374913d09e40da0b7ea6227560a6cd5d7903a)